src/scripts/orb.py

```diff
@@ -0,0 +1,145 @@
+"""
+# Purpose:
+
+Wait for the Shipyard environment to be ready,
+then put it's data in the environment.
+
+# Required environment variables:
+
+- BASH_ENV
+- CIRCLE_PROJECT_USERNAME
+- CIRCLE_PROJECT_REPONAME
+- CIRCLE_PROJECT_BRANCH
```

Comment:
These three are not currently required. They can be, if you like.

Reply:
They're required for the script to work, but since CircleCI injects, didn't add a check.